datasets/multi_eurlex/README.md

```diff
@@ -0,0 +1,309 @@
+
+```python
+from datasets import load_dataset
+dataset = load_dataset('multi_eurlex', 'en')
+```
+
+```json
+{
+  "celex_id": "31979D0509",
+  "text": "COUNCIL DECISION  of 24 May 1979  on financial aid from the Community for the eradication of African swine fever in Spain  (79/509/EEC)\nTHE COUNCIL OF THE EUROPEAN COMMUNITIES\nHaving regard to the Treaty establishing the European Economic Community, and in particular Article 43 thereof,\nHaving regard to the proposal from the Commission (1),\nHaving regard to the opinion of the European Parliament (2),\nWhereas the Community should take all appropriate measures to protect itself against the appearance of African swine fever on its territory;\nWhereas to this end the Community has undertaken, and continues to undertake, action designed to contain outbreaks of this type of disease far from its frontiers by helping countries affected to reinforce their preventive measures ; whereas for this purpose Community subsidies have already been granted to Spain;\nWhereas these measures have unquestionably made an effective contribution to the protection of Community livestock, especially through the creation and maintenance of a buffer zone north of the river Ebro;\nWhereas, however, in the opinion of the Spanish authorities themselves, the measures so far implemented must be reinforced if the fundamental objective of eradicating the disease from the entire country is to be achieved;\nWhereas the Spanish authorities have asked the Community to contribute to the expenses necessary for the efficient implementation of a total eradication programme;\nWhereas a favourable response should be given to this request by granting aid to Spain, having regard to the undertaking given by that country to protect the Community against African swine fever and to eliminate completely this disease by the end of a five-year eradication plan;\nWhereas this eradication plan must include certain measures which guarantee the effectiveness of the action taken, and it must be possible to adapt these measures to developments in the situation by means of a procedure establishing close cooperation between the Member States and the Commission;\nWhereas it is necessary to keep the Member States regularly informed as to the progress of the action undertaken,",
+  "labels": [
+    "192",
+    "2356",
+    "2560",
+    "862",
+    "863"
+  ]
+}
+```
+
+### Data Fields
+
+**Multilingual use of the dataset**
+
+The following data fields are provided for documents (`train`, `dev`, `test`):
+
+`celex_id`: (**str**)  The official ID of the document. The CELEX number is the unique identifier for all publications in both Eur-Lex and CELLAR.\
+`text`: (dict[**str**])  A dictionary with the 23 languages as keys and the full content of each document as values.\
+`labels`: (**List[str]**) The relevant EUROVOC concepts (labels).
```

Comment:
Is there a list with the mapping between the label id (int) and the label name (str) ?
It would be nice to have a link here to this mapping

Reply:
Sure, I updated the README, accordingly.